src/cljs/nnts2/events.cljs

```diff
@@ -1,27 +1,41 @@
 (ns nnts2.events
   (:require
-    [re-frame.core :as re-frame]
-    [nnts2.db :as db]
-    [nnts2.user.api :as user-api]))
+   [re-frame.core :as re-frame]
+   [day8.re-frame.http-fx]
+   [nnts2.db :as db]
+   [nnts2.organization.api :as org-api]
+   [nnts2.user.api :as user-api]))
 
 (re-frame/reg-event-db
-  ::initialize-db
-  (fn [_ _]
-    db/default-db))
+ ::initialize-db
+ (fn [_ _]
+   db/default-db))
 
 (re-frame/reg-event-db
-  ::set-active-panel
-  (fn [db [_ active-panel]]
-    (assoc db :active-panel active-panel)))
+ ::set-active-panel
+ (fn [db [_ active-panel]]
+   (assoc db :active-panel active-panel)))
 
 
 (re-frame/reg-event-fx
-  ::get-user-info
-  (fn [_ event]
-    (user-api/get-info)))
+ ::get-user-info
+ (fn [{db :db} event]
+   {:http-xhrio user-api/user-info-map
+    :db (assoc db :loading true)}))
 
+(re-frame/reg-event-fx
+ :create-organization
```

Comment:
I would recommend separating out module event handlers, subscription handlers, views etc in sub-directories.
Otherwise this events file will grow really large

Reply:
Yeah, makes sense. Did plan on doing this eventually, but now's a good time.